File: Assignment16/C/Program2.c
// PS - Accept number of rows and number of columns from user and display below pattern
// Row: 4
// Row: 3
// Output:
/* 
A 	B 	C 	D 	
a 	b 	c 	d 	
A 	B 	C 	D 	
a 	b 	c 	d 
*/

#include <stdio.h>

void Display(int Row, int Col)
{
    int i = 0, j = 0;
    char ch = '\0';

    for (i = 1; i <= Row; i++)
    {
        if ((i % 2) == 0)
        {
            for (j = 1, ch = 'a'; j <= Col; j++,ch++)
            {
                printf("%c \t", ch);
            }
        } else {
            for (j = 1, ch = 'A'; j <= Col; j++,ch++)
            {
                printf("%c \t", ch);
            }
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter Rows \n");
    scanf("%d", &iValue1);

    printf("Enter Col \n");
    scanf("%d", &iValue2);

    Display(iValue1, iValue2);

    return 0;
}

File: Assignment38/Java/Program2.java
// PS - write java program which accepts number of rows and number of columns from user and display below pattern
// A	B	C	D	
// a	b	c	d	
// A	B	C	D	
// a	b	c	d

import java.util.*;

class Pattern
{
    public int Arr[][];

    public Pattern(int i, int j)
    {
        Arr = new int[i][j];
    }

    public void CharPattern()
    {
        char ch = 'A';
        for(int i = 0; i < Arr.length; i++)
        {
            for(int j = 0; j < Arr[i].length; j++)
            {
                if((i % 2) != 0)
                {
                    System.out.print((char)(ch + 32)+"\t");
                }
                else{
                    System.out.print(ch+"\t");
                }
                ch++;
            }
            ch = 'A';
            System.out.println();
        }
    }
}

class Program2
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows");
        int iRows = sobj.nextInt();

        System.out.println("Enter number of columns");
        int iCols = sobj.nextInt();

        Pattern pobj = new Pattern(iRows, iCols);

        pobj.CharPattern();
    }
}